Empire/Empire.cpp

```diff
@@ -233,6 +233,7 @@ namespace {
       * Also checks if elements will be completed this turn. */
     void SetProdQueueElementSpending(
         std::map<std::set<int>, float> available_pp, float available_stockpile,
+        float& stockpile_transfer,
```

Comment:
Not a fan of having a random parameter in the middle also act as an output while none of the others do. Perhaps rework to return the amount to transfer?